src/styles/global.scss

```diff
@@ -38,7 +38,7 @@ $red: #fc5b5b;
   --spacing-l: calc(var(--base-spacing) * 3);
   --spacing-xl: calc(var(--base-spacing) * 4);
   --spacing-xxl: calc(var(--base-spacing) * 5);
-  --max-width: calc(35 * var(--spacing-m));
+  --max-width: calc(80 * var(--spacing-m));
```

Comment:
We might want to make new variables. Something like `--narrow-column` and `--two-column` or something. Changing `max-width` has broken the `container` class, which means that anything with a `container` class now looks pretty bad. Like, check out the home page right now!